src/dmd/todt.d

```diff
@@ -1389,6 +1389,9 @@ private extern (C++) class TypeInfoDtVisitor : Visitor
         }
 
         // xgetRTInfo
+        if(!sd.getRTInfo)
+            error(sd.loc, "ICE: RTInfo not evaluated for %s", sd.toChars());
```

Comment:
It's either an ICE or an error.  Pick one for the sake of coverage. :-)

Reply:
It is an ICE, but I was thinking of making it a warning instead, because it is likely to show up on user code that doesn't care about RTInfo (yet).